client/src/components/EditProfile/ProfilePhotoTab/ProfilePhotoTab.tsx

```diff
@@ -3,24 +3,46 @@ import { Typography, Box, Grid, Avatar, Button, InputLabel } from '@material-ui/
 import AccountCircleIcon from '@material-ui/icons/AccountCircle';
 import CloudUploadIcon from '@material-ui/icons/CloudUpload';
 import DeleteIcon from '@material-ui/icons/Delete';
+import { useAuth } from '../../../context/useAuthContext';
+import { useSnackBar } from '../../../context/useSnackbarContext';
+import uploadImage from '../../../helpers/APICalls/uploadImage';
+import deleteImage from '../../../helpers/APICalls/deleteImage';
 import useStyles from './useStyles';
 
 const ProfilePhoto = (): JSX.Element => {
   const classes = useStyles();
+  const { loggedInUser } = useAuth();
+  const { updateSnackBarMessage } = useSnackBar();
   const [displayPhoto, setDisplayPhoto] = useState<string>('');
   const [isUploaded, setIsUploaded] = useState<boolean>(false);
   const [isWrongFileType, setIsWrongFileType] = useState<boolean>(false);
 
   const onPhotoUploadChange = (e: React.ChangeEvent<HTMLInputElement>): void => {
     const file: File = (e.target.files as FileList)[0];
     const fileURL = file ? URL.createObjectURL(file) : '';
+    const images = [];
     if (file && (file.type === 'image/jpeg' || file.type === 'image/png' || file.type === 'image/jpg')) {
       setIsUploaded(true);
       setIsWrongFileType(false);
       setDisplayPhoto(fileURL);
+      if (loggedInUser) {
+        images.push(file);
+        uploadImage(images, loggedInUser.profile).then((data) => {
+          if (data.error) updateSnackBarMessage('An error occurred while uploading the image.');
```

Comment:
Good work making these small changes but resort to making them in a separate PR :)